kuma/wiki/content.py

```diff
@@ -228,8 +228,19 @@ def get_seo_description(content, locale=None, strip_markup=True):
                 content = summary_section
 
         # Need to add a BR to the page content otherwise pyQuery wont find
-        # a <p></p> element if it's the only element in the doc_html
-        seo_analyze_doc_html = content + '<br />'
+        # a <p></p> element if it's the only element in the doc_html.
+
+        # Note, PyQuery is magically clumsy in that it will try download
+        # it if the first and only argument looks like a URL.
+        # It does that my looking for args[0] being a string and
+        # containing 'http://' or 'https://'.
+        # Let's pre-emptively make that same check ourselves and if this
```

Comment:
Nit: the comment, which talks about performing the same check as `PyQuery`, is out-of-sync with the final fix of simply adding a space.